roles/openshift_prometheus/templates/node-exporter.j2

```diff
@@ -0,0 +1,49 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: prometheus-node-exporter
+  labels:
+    app: prometheus-node-exporter
+    role: monitoring
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: prometheus-node-exporter
+        role: monitoring
+      name: prometheus-exporter
+    spec:
+      serviceAccountName: prometheus-node-exporter
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - image: "{{ l_openshift_prometheus_node_exporter_image_prefix }}prometheus-node-exporter:{{ l_openshift_prometheus_node_exporter_image_version }}"
```

Comment:
@pgier can you confirm we'll be releasing the image under this name? e.g. `registry.access.redhat.com/openshift3/prometheus-node-exporter:<version>`

Reply:
Yes, that's the correct name